src/mlpack/methods/logistic_regression/logistic_regression_impl.hpp

```diff
@@ -87,7 +87,8 @@ double LogisticRegression<MatType>::Train(
       lambda);
 
   // Set size of parameters vector according to the input data received.
-  parameters = arma::rowvec(predictors.n_rows + 1, arma::fill::zeros);
+  if (parameters.n_elem != predictors.n_rows + 1)
+    parameters = arma::rowvec(predictors.n_rows + 1, arma::fill::zeros);
```

Comment:
Couldn't we remove the line below? `InitialPoint()` is not used inside the optimizer or the LR class.

Reply:
You're right!  I removed it entirely in c976767.  Thanks for pointing that out.